Library/Homebrew/test/test_keg.rb

```diff
@@ -304,4 +304,35 @@ def test_removes_broken_symlinks_that_conflict_with_directories
     keg.unlink
     keg.uninstall
   end
+
+  def test_mach_o_files_skips_hardlinks
+    a = (HOMEBREW_CELLAR/"a")/"1.0"
```

Comment:
You only need the parentheses if you need to invoke a method on the `Pathname` instance, like in `(a/"lib").mkpath`. In this case, the expression is evaluated from left-to-right anyway, so the parentheses don't provide any benefit. Though that's only of secondary importance, as we don't usually chain `Pathname#/` invocations and instead prefer to use something like this:

```ruby
a = HOMEBREW_CELLAR/"a/1.0"
```

(That means all the fixed trailing path components are in a single string instead of incrementally appending individual path components.)